app/controllers/samples_controller.rb

```diff
@@ -277,6 +277,31 @@ def dimensions
     end
   end
 
+  def stats
+    domain = params[:domain]
```

Comment:
I see params used directly a lot in this controller but should probably whitelist and use sample_params?

Reply:
Discussed offline.
Apparently, the strong parameters just filter out parameters and log that, letting the action proceed as normal, but without the parameters.
It looks as if this is more useful for write actions, where we might want to pass the parameters directly to create and/or update actions.
If misconfigured, the effect of strong parameters might actually be worse than attempts to tamper it.